docs/docker_walkthrough.rst

```diff
@@ -0,0 +1,539 @@
+  └── data              <-- the `data` directory which will contain your training data
+      └── stories.md    <-- the file which contains your stories
+
+For a start let's build a bot which can say `Hi`, asks for your mood and then
+gives a different answer depending on your input. Put these basic stories
+in the file ``data/stories.md``:
+
+.. code-block:: md
+
+  ## happy_path
+  * greet
+    - utter_greet
+  * mood_happy
+    - utter_happy
+  * goodbye
+    - utter_goodbye
+
+  ## sad_path
+  * greet
+    - utter_greet
+  * mood_unhappy
+    - utter_cheer_up
+  * goodbye
+    - utter_goodbye
+
+2.2 Create the Bot's Universe
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After defining some training data for our bot, we have to define its domain.
+You can think about the domain as the world your bots live in and contains all
+capabilities your bots has.
+
+To handle our examples stories, our bot has to have the following capabilites:
+
+  - recognize user intents
+
+    - saying hello to the bot (`greet`)
+    - saying that they are happy (`mood_happy`)
+    - saying that they are sad (`mood_unhappy`)
+    - saying goodbye to the bot (`good_bye`)
+
+  - respond to the user input
+
+    - response for happy users (`utter_happy`)
+    - cheer up a sad user (`utter_cheer_up`)
+    - say goodbye (`utter_goodbye`)
+
+.. note::
+
+  By convention the templates for bot responses have the prefix ``utter_``.
+  You can follow this convention but don't have to.
+
+The domain file is used to stores all these capabilities.
+Further, we put templates for our bot's responses in it.
+
+Let's start by creating the domain file:
+
+.. code-block:: bash
+
+  touch domain.yml
+
+Then we put our required capabilities in the created ``domain.yml``:
+
+.. code-block:: yaml
+
+    intents:            # <-- intents of the user speaking to your bot
+      - greet
+      - mood_happy
+      - mood_unhappy
+      - goodbye
+
+    actions:            # <-- actions your bot can execute in response to user input
+      - utter_greet
+      - utter_happy
+      - utter_cheer_up
+      - utter_goodbye
+
+    templates:          # <-- templates for your bot's answers
+      utter_greet:
+        - text: "Hi, how is it going?"
+      utter_happy:
+        - text: "Great, carry on!"
+      utter_cheer_up:
+        - text: "Here is something to cheer you up:"
+          image: "https://i.imgur.com/nGF1K8f.jpg"
+      utter_goodbye:
+        - text: "Goodbye!"
+
+2.3 Training our Bot
+~~~~~~~~~~~~~~~~~~~~
+
+We now have everything set up to train our bot! Hooray!
+To so, we have to mount the training data and the domain file in the Rasa Core
+container.
+
+.. code-block:: bash
+
+  docker run \
+    -v $(pwd):/app/project \
+    -v $(pwd)/models/rasa_core:/app/models \
+    rasa/rasa_core:latest \
+    train \
+      --domain project/domain.yml \
+      --stories project/data/stories.md \
+      --out models
+
+Yey! You just trained your chatbot for the first time!
```

Comment:
I think there are too many excitement words included :) For documentation, I would suggest keeping documentation content a bit more toned down because developers should use the docs as a concise and guide of how to use the software.

Reply:
Also, I think it's better to leave out 'for the first time' part, because the quickstart is the place where we expect people to build their first Rasa assistant.